gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/util/ResourceReferenceUtilTest.java

```diff
@@ -35,14 +35,14 @@
     private static final int FOUR = 4;
 
     @Mock
-    private ImmunizationChecker immunizationChecker;
+    private DatabaseImmunizationChecker immunizationChecker;
 
```

Comment:
I think you could have `@InjectMocks ResourceReferenceUtil` declaration and then wouldn't need to create the ResourceReferenceUtil object in each test.